tidy3d/components/medium.py

```diff
@@ -229,13 +263,20 @@ def complex_fields(self) -> bool:
 class TwoPhotonAbsorption(NonlinearModel):
     """Model for two-photon absorption (TPA) nonlinearity which gives an intensity-dependent
     absorption of the form :math:`\\alpha = \\alpha_0 + \\beta I`.
+    Also includes free-carrier absorption (FCA) and free-carrier plasma (FCP) effects.
```

Comment:
Is FCP the same as free-carrier dispersion (i.e. modulation of the real part of the permittivity)? I've heard the latter but not the former.

Reply:
yeah, if that is more familiar I can change the terminology